.circleci/config.yml

```diff
@@ -1386,9 +1389,9 @@ jobs:
       - run:
           name: test:e2e:chrome:mmi
           command: |
-            TESTFILES=$(circleci tests glob "test/e2e/mmi/**/*.spec.ts")
+            TESTFILES=$(circleci tests glob "test/e2e/playwright/mmi/**/*.spec.ts" | circleci tests split)
```

Comment:
Hmm, why was `circleci tests split` added here? The docs for it seem to suggest that it's not recommended anymore:

https://circleci.com/docs/parallelism-faster-jobs/#the-tests-split-command



>The circleci tests split command has been superceded by circleci tests run as this command also gives you access to the [rerun failed tests](https://circleci.com/docs/rerun-failed-tests/) options.

Reply:
Good catch. It was not needed so I removed it